src/main/java/useCases/Calculator/TotCalculator.java

```diff
@@ -0,0 +1,80 @@
+package useCases.Calculator;
+
+import entities.Event;
+import entities.Task;
+import entities.User;
+import useCases.Calculator.helper.CalculateLocalDateTime;
+import useCases.Calculator.helper.TimeConverter;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class TotCalculator extends Calculator{
+    public TotCalculator(){}
+
+    /**
+     * Return the total amount of time the user had scheduled during the given time interval
+     * @param user the user we examine
+     * @param unit the unit of time (Day/Week/Month/Year)
+     * Precondition: unit must be "month"/"week"/"day". Other strings are not considered as valid units.
+     *
+     */
+    public int rawTime(User user, String unit) {
+        List<Event> events = super.gather_events(user, unit);
+        int acc = 0;
+        for (Event event : events) {
+            CalculateLocalDateTime c = new CalculateLocalDateTime(event.getStartTime(), event.getEndTime());
+            int diff = 0;
+            diff = diff + c.day_diff();
```

Comment:
I'm not sure if doing a day difference would makes sense for events. 
Personally, my events don't span more than a day (like lectures, doing an assignment, meetings), so I think calculating hours, or even minutes (maybe both) would be more appropriate.